CONTRIBUTING.md

```diff
@@ -266,6 +266,17 @@ the production server.
 To see deploy logs and more, visit project's [dashboard][] -- Netlify login
 required.
 
+## Approver and Maintainer practices
+
+This last section includes guidelines and some common practices used by approvers and maintainers while doing code reviews:
+
+* PRs with changes to documentation co-owned by a SIG (collector, demo, language-specific...) should aim for two approvals: one by a docs approver and one by a SIG approver:
+  * Doc approver label such PRs with `sig:<name>` and tag the SIG `-approvers` group on that PR
+  * If no SIG approval is given within a certain grace period, docs maintainer may use their own judgement to merge that PR
```

Comment:
Like two weeks? Can we add a sample period?

Reply:
Yes, let's go with two weeks.

Reply:
I kept it out on purpose to leave this to our judgement, but I can say something like "2 weeks in general", allowing us to wait longer (probably never the problem), but also merge somthing urgent quicker without getting in discussions